.github/workflows/build-docker.yml

```diff
@@ -31,6 +56,7 @@ jobs:
         with:
           push: true
           file: "./tools/releasing/packaging/docker/Dockerfile"
-          tags: ${{ env.docker_username }}/python-bindings:${{ env.branch }},${{ env.docker_username }}/python-bindings:latest
+          tags: ${{ env.docker_username }}/python-bindings:${{ env.TAG }}
```

Comment:
This is the main change / fix.
Now, if the branch is `master`, the tag applied is `latest`. Otherwise, it is the name of the branch / Git reference.